internal/provisioners/default/zz-default.provisioners.yaml

```diff
@@ -798,3 +798,120 @@
         selector:
           app.kubernetes.io/instance: {{ .State.service }}
         type: ClusterIP
+
+- uri: template://default-provisioners/mssql
+  type: mssql
+  init: |
+    randomDatabase: db-{{ randAlpha 8 }}
+    randomPassword: {{ randAlphaNum 16 | quote }}
+  state: |
+    service: mssql-{{ .SourceWorkload }}-{{ substr 0 8 .Guid | lower }}
+    database: {{ dig "database" .Init.randomDatabase .State | quote }}
+    username: sa
+    password: {{ dig "password" .Init.randomPassword .State | quote }}
+  outputs: |
```

Comment:
I understand why we have 4 distinct `outputs` here given the `sqlcmd` example taking separate parameters.

But please could you also add a more generic one `connection`, in this case that will return:
```
connection: "Server=tcp:{{ .State.service }},1433;Initial Catalog={{ .State.database }};User="
```

There are multiple benefits with this `connection` field:
- Abstracting from the Devs about how to build the connection string
- There are more fields in other scenario for example with Azure SQL: `"Server=tcp:server.fully_qualified_domain_name,1433;Initial Catalog=database_name;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;Authentication=Active Directory Default";User ID=administrator_login;Password=administrator_login_password"`
- As Dev, you don't have to change your Score file if you point to an in-cluster database or a managed database, etc.

Reply:
Connection output variable was added